docs/src/components.js

```diff
@@ -136,6 +137,7 @@ const Paragraph = (p) => <Text.p {...p} />
 
 const components = {
   ...DS,
+  ...icons,
```

Comment:
@craigpalermo updated to use the full/star import! Couldn't find any conflicts with icons after going through all the docs after the change. 👍